clients/TypeScript/packages/client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cardano-ogmios/client",
-  "version": "0.1.2",
+  "version": "3.2.0",
```

Comment:
I wanted to propose that some time ago but then, thought it may be a good idea to keep the clients and the server on their own versioning. There may be new versions of the client(s) without any new changes from the server :thinking:. Unless we treat everything in the repository (client(s) + server) as _one_ "ogmios" entity and everything is released at the same time. 

Incidentally, I asked myself this question again yesterday while finalizing the changelog for the release. There's currently no changelog for any of the client package. Here again, if decide to move all modules in lockstep then I'd also be in favor of keeping _one_ global changelog and have symlinks from each client package root to that global changelog.

Reply:
> may be a good idea to keep the clients and the server on their own versioning.

That was also my initial thought, but considering we're in the same code repository it makes sense to release at the same time. If we need separation, then lifting the client to a new repo (under and org space) is the way to go IMO.

Reply:
Having said that, I do feel a single repo has a number of advantages:

1. The schema package, which is fully derived from the JSON-WSP schema, is then simultaneously available with server releases.
2. Ogmios REPL is the universal quick start UI, not just a TypeScript-focused product.
3. The client is used to provide integration test coverage.

Reply:
I agree with all your points above :+1: 
Would you mind doing --> 

> have symlinks from each client package root to that global changelog. 

Pretty please :pray: